spec/features/dinosaur/user_can_update_dinosaurs_spec.rb

```diff
@@ -48,7 +75,7 @@
     end
 
     scenario 'form is pre-filled with current values' do
-      dinosaur = FactoryGirl.create(:dinosaur)
+      dinosaur = Dinosaur.last
```

Comment:
I would not recommend using `Dinosaur.last`. What made you move away from FactoryGirl?